.github/workflows/mandatory-conformance.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Github action job to test core java library features on
+# downstream client libraries before they are released.
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+name: mandatory-conformance
+jobs:
+  mandatory-conformance:
```

Comment:
This file is just like [the changes in this PR](https://github.com/googleapis/nodejs-bigtable/pull/1349/files#diff-7bfd467904d753bc34e8f91ffbb8165dc048bbe3610af168c600072d0ec473c0R23), but with `conformance` renamed to `mandatory-conformance`.